.nvmrc

```diff
@@ -0,0 +1 @@
+v16.13.0
```

Comment:
I added this because I was having trouble installing the app on my linux computer with an older version of node. Now, if you are using nvm, you can just do `nvm use` to use this version of node.